Support for Angular Material 9

Hi.

Trying to compile with Angular Material 9 (exactly `9.0.0-rc.7`), I get errors looking like this :

```
import { CanUpdateErrorStateCtor, ErrorStateMatcher } from '@angular/material';
                                                           ~~~~~~~~~~~~~~~~~~~
../node_modules/ngx-material-file-input/lib/file-input/file-input.component.d.ts:3:35
error TS2306: File '(...)/node_modules/@angular/material/index.d.ts' is not a module.
```

It seems related to the fact that from version 9, Angular Material does not export all its classes in `index.d.ts` anymore, and one has to import them from deeper paths, for ex:
```
import { ErrorStateMatcher } from "@angular/material/core"
```

It's hopefully not such a big deal to update, maybe you have plans for this ?

Thank you, the file input is great by the way !
Mathias

--- a/apps/demo/src/app/utils/example-error-state-matcher.ts
+++ b/apps/demo/src/app/utils/example-error-state-matcher.ts
@@ -1,4 +1,4 @@
-import { ErrorStateMatcher } from '@angular/material';
+import { ErrorStateMatcher } from '@angular/material/core';
 import { FormControl, NgForm, FormGroupDirective } from '@angular/forms';
 
 /**

--- a/libs/material-file-input/src/lib/file-input/file-input-mixin.ts
+++ b/libs/material-file-input/src/lib/file-input/file-input-mixin.ts
@@ -1,5 +1,5 @@
 import { FormGroupDirective, NgControl, NgForm } from '@angular/forms';
-import { CanUpdateErrorStateCtor, ErrorStateMatcher, mixinErrorState } from '@angular/material';
+import { CanUpdateErrorStateCtor, ErrorStateMatcher, mixinErrorState } from '@angular/material/core';
 
 // Boilerplate for applying mixins to FileInput
 /** @docs-private */

--- a/libs/material-file-input/src/lib/file-input/file-input.component.spec.ts
+++ b/libs/material-file-input/src/lib/file-input/file-input.component.spec.ts
@@ -1,6 +1,6 @@
 import { async, ComponentFixture, TestBed, tick, fakeAsync } from '@angular/core/testing';
 import { FormsModule, NG_VALUE_ACCESSOR, NgControl, ReactiveFormsModule, FormControl, FormGroupDirective, NgForm } from '@angular/forms';
-import { ErrorStateMatcher } from '@angular/material';
+import { ErrorStateMatcher } from '@angular/material/core';
 import { MatButtonModule } from '@angular/material/button';
 import { MatFormFieldModule } from '@angular/material/form-field';
 import { MatIconModule } from '@angular/material/icon';

--- a/libs/material-file-input/src/lib/file-input/file-input.component.ts
+++ b/libs/material-file-input/src/lib/file-input/file-input.component.ts
@@ -1,6 +1,6 @@
 import { Component, OnInit, Input, ElementRef, OnDestroy, HostBinding, Renderer2, HostListener, Optional, Self, DoCheck } from '@angular/core';
 import { ControlValueAccessor, NgControl, NgForm, FormGroupDirective } from '@angular/forms';
-import { ErrorStateMatcher } from '@angular/material';
+import { ErrorStateMatcher } from '@angular/material/core';
 import { MatFormFieldControl } from '@angular/material/form-field';
 import { FocusMonitor } from '@angular/cdk/a11y';
 import { coerceBooleanProperty } from '@angular/cdk/coercion';

